devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install RPM packages
+  dnf:
+      name: ['git', 'vim', 'poetry', 'python3-pip', 'freeipa-server']
+      state: present
+
+- name: install python deps with poetry
+  shell: poetry install
+  become: yes
+  become_user: vagrant
+  args:
+    chdir: /vagrant/
+
+- name: install freeipa server
+  shell: ipa-server-install -a adminPassw0rd! --hostname=ipa.fas2ipa.test -r FAS2IPA.TEST -p adminPassw0rd! -n fas2ipa.test -U
+
+- name: get freeipa-fas
+  git:
+    repo: https://github.com/fedora-infra/freeipa-fas.git
+    dest: /tmp/freeipa-fas
```

Comment:
I'd rather store this in `/home/vagrant/freeipa-fas`, files in /tmp get deleted after a while.

Reply:
np. we also do it this way on noggin too. so will fix there as well